rust_snuba/src/mutations/clickhouse.rs

```diff
@@ -170,6 +170,76 @@ fn format_query(table: &str, batch: &MutationBatch) -> Vec<Vec<u8>> {
     ]
 }
 
+struct ClickhouseTestClient {
+    url: String,
+    table: String,
+    client: Client,
+}
+
+impl ClickhouseTestClient {
+    pub async fn new(table: String) -> anyhow::Result<Self> {
+        // hardcoding local Clickhouse settings
+        let hostname = "127.0.0.1";
+        let http_port = 9000;
+        let url = format!("http://{hostname}:{http_port}");
+
+        let client = reqwest::Client::new();
+        let test_table = format!("{table}_test_local\n");
```

Comment:
this should contain a UUID or otherwise random value. we will have many tests that use this client, and `cargo test` runs them in parallel. if the tests were run sequentially, it would not be necessary to create a separate table per test at all

Reply:
Oh, I didn't realize that `cargo test` would run them in parallel. Thanks for calling that out

Reply:
Done